src/forms.css

```diff
@@ -59,6 +59,7 @@
 }
 .input[type='search'] {
   appearance: textfield;
+  -webkit-appearance: none;
```

Comment:
`appearance` and `-webkit-appearance` clash here: `-webkit-appearance` is populated by Autoprefixer when we use `appearance`. So if you think `none` is a better value, we should just use `appearance: none`.